week7/src/components/Header.js

```diff
@@ -0,0 +1,34 @@
+import logoImg from "../assets/Linkbrary.png";
+import "./Header.css";
+import Folder from "./Folder.js";
+import { useMediaQuery } from "react-responsive";
+const profileAccount = ({ user }) => {
+  if (!user) {
+    return false;
+  }
+  const { email, profileImageSource } = user;
+  return (
+    <div className="profile-info">
+      <img className="logo" src={profileImageSource} alt="프로필 이미지" />
+      <div>{email}</div>
+    </div>
+  );
+};
+function Header({ user, folderName, owner }) {
+  const isTablet = useMediaQuery({ maxWidth: 1199 });
+  return (
+    <>
```

Comment:
빈 fragment를 넣기보다 `header` 태그를 이용해서 좀 더 시멘틱한 마크업을 해주세요

```suggestion
    <header>
```